public/catch-js-errors.js

```diff
@@ -0,0 +1,17 @@
+function sendErrorsToAPI( message, scriptUrl, lineNumber, columnNumber, error ) {
```

Comment:
I wonder if we should think about some kind of debouncer / filter here.
From what I see in `window.onerror` documentation, this can also be triggered by 'other' scripts errors ( like plugins for example )
So, if we or plugin creator somehow generates errors in a loop, than we can both kill the API or the app by sending so many post requests.
Maybe we can 'batch' it in some way?